Integrations/MailListener-POP3/MailListener-POP3.yml

```diff
@@ -0,0 +1,60 @@
+commonfields:
+  id: MailListener-POP3
+  version: -1
+name: MailListener-POP3
+display: MailListener-POP3
+category: Messaging
+description: Integration Template
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Server URL (e.g. https://example.com)
+  name: server
+  defaultvalue: pop.gmail.com
+  type: 0
+  required: true
+- display: Use system proxy
```

Comment:
reorder the parameter list:
1) server
2) port
3) email
4) password
5) use ssl
6) proxy
7) fetch incident
8) first fetch